ui/src/dashboards/actions/index.js

```diff
@@ -111,26 +115,45 @@ export const editCellQueryStatus = (queryID, status) => ({
   },
 })
 
+export const tvSelected = (dashboardID, tvID, values) => ({
+  type: TEMPLATE_VARIABLE_SELECTED,
+  payload: {
+    dashboardID,
+    tvID,
+    values,
+  },
+})
+
 // Stub Template Variables Data
 
-const templates = [
+const tempVars = [
   {
     id: 1,
     type: 'query',
     label: 'test query',
-    code: '$HOSTS',
+    tempVar: '$REGION',
     query: {
       db: 'db1.rp1',
       text: 'SHOW TAGS WHERE HUNTER = "coo"',
     },
-    values: ['h1', 'h2', 'h3'],
+    values: [
+      {value: 'us-west', type: 'tagKey'},
+      {value: 'us-east', type: 'tagKey'},
+      {value: 'us-mount', type: 'tagKey'},
+    ],
+    selectedValues: [{value: 'us-east', type: 'tagKey'}],
```

Comment:
This feels like redundant data.  Could this be part of the the `values`?  ... something like 
```
    values: [
      {value: 'us-west', type: 'tagKey'},
      {value: 'us-east', type: 'tagKey', selected: true},
      {value: 'us-mount', type: 'tagKey'},
    ]
```